### PR TITLE
Add payment method link has moved

### PR DESCRIPTION
**Pull request by lornajane**
## Description

I noticed a few broken links in the number/account guides, mostly pointing to the moved "Add a Payment Method" section. This fixes those links and also removes a repeated content section that sort of made sense but looked odd.

### CODE REVIEW

File: _documentation/numbers/guides/numbers.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ To rent a Nexmo virtual number:
 4. Click the number you want and validate your purchase.
 5. Your virtual number is now listed in **Your numbers**.
 
-If your account has no credit your virtual numbers are released for resale. To avoid this, enable [auto-reload payments](#add_payment_method).
+If your account has no credit your virtual numbers are released for resale. To avoid this, enable [auto-reload payments](/numbers/guides/payments#auto-reload-your-account-balance).
```
Comment (marklewin) [inline _documentation/numbers/guides/numbers.md]: This doesn't go to the "Auto reload your account balance" section for me, although the link looks correct?

Comment (lornajane) [reply _documentation/numbers/guides/numbers.md]: Mmm, I am not sure either. When I refresh, the link jumps down as you'd expect.  @mheap any advice for us?

Comment (mheap) [reply _documentation/numbers/guides/numbers.md]: This is another turbolinks issue I think. This is fine to merge, and I'll take a look at the deep link issue along with all the other turbolinks issues

Comment (marklewin) [reply _documentation/numbers/guides/numbers.md]: Thanks @mheap - merging.

**Pull request merged**